moskito-core/src/main/java/net/anotheria/moskito/core/producers/AbstractStats.java

```diff
@@ -65,7 +65,7 @@
 	/**
 	 * Contains all stat values that are part of this StatsObject. This is used to properly destroy them.
 	 */
-	private LinkedList<StatValue> statValuesList = new LinkedList<StatValue>();
+	private Collection<StatValue> statValuesList = new LinkedList<>();
```

Comment:
Why to change direct instantiation to abstract type? You know that using abstract type is twice as slow as the real type?